src/complex.rs

```diff
@@ -514,11 +520,9 @@ mod test {
     }
 
     #[test]
-    #[should_panic]
-    #[ignore]
     fn test_inv_zero() {
         // FIXME #5736: should this really fail, or just NaN?
```

Comment:
While we're here, that's referencing a rust-lang/rust issue; can you please update this to #20?